rpcd/etc/openstack_deploy/user_apt_packages.yml

```diff
@@ -0,0 +1,11 @@
+# This file will contain any apt packages that the deployer would like to pin to a particular version.
```

Comment:
openstack-ansible bails for me when this file is empty (as it is by default), not sure if this is an ansible bug or what?